spec/lib/phony/countries_spec.rb

```diff
@@ -797,7 +797,7 @@ def self.it_splits number, expected
       it_splits '299691123', ['299', '691', '123']
     end
     describe 'Guadeloupe (French Department of)' do
-      it_splits '590123456789', %w(590 123 456 789)
+      it_splits '590590456789', %w(590 590 45 67 89)
```

Comment:
All local phone numbers are written +590 590 xx xx xx in Guadeloupe, and all mobile phone numbers are written +590 690 xx xx xx.
I have never seen another format so far. From France, people can either type 0590 xx xx xx or 00590 590 xx xx xx or 0033 590 xx xx xx.